### ISSUE TITLE
Laser curls back inward when moved slowly out of Inspector edge

### ISSUE THREAD

**Issue by H3BO3** [Frooxius/NeosPublic#492]
Position your laser close to the edge of the properties pane, and slowly move it right like you intend to point it past the panel. The laser curls back in to the left a certain amount, highlighting buttons and fields before finally breaking out of the panel

**Comment by shiftyscales** [comment]
This will likely be fixed with the addition of adjustable laser settings: #487 
In particular, adjusting the `StickThreshold` will help prevent those scenarios from occurring.

If you'd like- you can try adjusting the values of the stick threshold on the InteractionLaser component, and see if there's a value that feels better to you, @H3BO3.

**Comment by H3BO3** [comment]
No, the tip of the laser literally slides back inward as you move the laser out, it doesn't stick on the edge as you move outward. I had a lot of trouble clicking component deletion buttons because this kept happening.

**Comment by shiftyscales** [comment]
Any chance you could capture a quick clip of this? I think it would help a lot to show it. I'm having a tough time visualizing it myself.

**Comment by H3BO3** [comment]
[https://streamable.com/9n1led](url) Captured a clip - It's not so much that it always scoots away from the edges, now that I try it some more, but the edge behavior seems very inconsistent. Sometimes it scoots back, sometimes it hangs right before the edge, sometimes it works normally, etc

**Comment by Frooxius** [comment]
That looks like it's just trying to snap to the edge of the panel since you're moving your cursor past the edge. I don't see anything inherently wrong with the behavior, adjusting the sticking threshold would help in this case or making the inspector collider bigger. The panels will be redesigned too later so that could be considered.

**Comment by H3BO3** [comment]
You can clearly see that around 0:55 the laser tip slides back inward towards the buttons instead of sticking consistently on the edge
